ngraph/core/include/openvino/runtime/tensor.hpp

```diff
@@ -50,10 +50,12 @@ class OPENVINO_API Tensor {
     friend class ov::runtime::VariableState;
 
 public:
+    using Ptr = std::shared_ptr<Tensor>;
```

Comment:
Why wee need Ptr?

Reply:
The old evaluate methods work with vector of pointers to tensor. In new API I use the same approach.
Tensor::Ptr is just a shortcut which we also have for some other structures.

Reply:
`Tensor` itself is already smart pointer, we don't need additional pointer around pointers